### PR TITLE
can delete bookingpackages lines with not unique package_id

### PR DESCRIPTION
**Pull request by gaetanHerve**

### CODE REVIEW

**Comment by gaetanHerve** [comment]
I'm gonna refactor Bookingsupsinfo, Bookingcalquantities and Bookingpackages controllers using a parent class. Most of the mechanics are the same

**Comment by osallou** [comment]
Warning: in packages we could have only one element per paclage id, this may not be the same for otherd

**Comment by gaetanHerve** [comment]
yes, I'll be careful to keep the right behaviour

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## 2.5.0
 
+* [booking_quantities] avoid having two qtes with the same name for the same resource
```
Comment (osallou) [inline CHANGELOG.md:5]: will be 2.6.0 now

File: Modules/booking/Controller/BookingsupsabstractController.php [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        return false;
+    }
+
+    protected function hasInvoicingUnitsDuplicates($supResources, $supIsInvoicingUnit, $id_space, $lang) {
+        $result = false;
+        $invoicingUnitsResources = [];
+        foreach ($supResources as $index => $resource) {
+            if ($supIsInvoicingUnit[$index] == 1) {
+                if (in_array($resource, $invoicingUnitsResources)) {
+                    $_SESSION["flash"] = BookingTranslator::maxInvoicingUnits($lang);
```
Comment (osallou) [inline Modules/booking/Controller/BookingsupsabstractController.php:159]: is message adapted to package/supp/....?

Comment (gaetanHerve) [reply Modules/booking/Controller/BookingsupsabstractController.php:159]: $invoicable property is set to false for supplementaries and packages. So it never pass through hasInvoicingUnitsDuplicates() (and these lines of code). cf line 101:
```
if ($this->invoicable && $this->hasInvoicingUnitsDuplicates($supResources, $supIsInvoicingUnit, $id_space, $lang))
```

**Comment by osallou** [comment]
changelog conflict, else, should it be merged?

**Pull request merged**